### DISCUSSION TITLE
Issue - ACF group fields are not showing in the dynamic fields. #9571

### DISCUSSION THREAD

**Discussion by ZoltanGalCMDesign** [elementor/elementor#23603] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

As described in this issue from 2019: Bug: ACF group fields are not showing in the dynamic fields. #9571
I was wondering if this feature would make its way to Elementor in the near future.

Please vote it up if You need this or would like to use it.

Thank You

### Describe the solution you'd like

It would be nice if we can use ACF group field's subfields in the dynamic fields.

**Comment by danielvigh** [comment]
+1

**Comment by TakacsAdrian** [comment]
+1
